cmd/geth/main.go

```diff
@@ -148,29 +148,41 @@ var (
 		utils.FakePoWFlag,
 		utils.NoCompactionFlag,
 		utils.GpoBlocksFlag,
+		utils.GpoLegacyBlocksFlag,
 		utils.GpoPercentileFlag,
+		utils.GpoLegacyPercentileFlag,
 		utils.EWASMInterpreterFlag,
 		utils.EVMInterpreterFlag,
 		configFileFlag,
 	}
 
 	rpcFlags = []cli.Flag{
-		utils.RPCEnabledFlag,
-		utils.RPCListenAddrFlag,
-		utils.RPCPortFlag,
-		utils.RPCCORSDomainFlag,
-		utils.RPCVirtualHostsFlag,
+		utils.HTTPEnabledFlag,
+		utils.HTTPListenAddrFlag,
+		utils.HTTPPortFlag,
+		utils.HTTPCORSDomainFlag,
+		utils.HTTPVirtualHostsFlag,
+		utils.RPCLegacyEnabledFlag,
+		utils.RPCLegacyListenAddrFlag,
+		utils.RPCLegacyPortFlag,
+		utils.RPCLegacyCORSDomainFlag,
+		utils.RPCLegacyVirtualHostsFlag,
```

Comment:
A suggestion, and I'm not sure if everyone agrees (@karalabe?), but wouldn't it make things easier on the eyes if we have `XxxFlag` and `LegacyXxxFlag` ? 
So we'd have `RPCVirtualHostsFlag` and `LegacyRPCVirtualHostsFlag`, consistently across the board?

Reply:
> Generally looks good to me, but I am going to test it a bit too. Also, I'm wondering if it wouldn't make sense to move out all legacy flags into `cmd/utils/flags_legacy.go`, and note the date or release that they were deprecated. That might make it easier later on to say "yeah this is deprecated since three years, let's delete this now"

I think this is a good idea. I was trying to follow the other deprecated flag syntax with `XXXLegacyFlag` but `Legacy` as a prefix seems much nicer. Would it be okay to note the date/release when they were deprecated in the usage message? Or as a comment in the code?